bodhi/server/consumers/masher.py

```diff
@@ -1027,12 +1029,21 @@ def __init__(self, tag, outputdir, comps, previous, log):
         self.tag = tag
         self.log = log
         self.success = False
-        mash_cmd = 'mash -o {outputdir} -c {config} -f {compsfile} {tag}'
-        mash_conf = config.get('mash_conf')
-        if os.path.exists(previous):
-            mash_cmd += ' -p {}'.format(previous)
-        self.mash_cmd = mash_cmd.format(outputdir=outputdir, config=mash_conf,
-                                        compsfile=comps, tag=self.tag).split()
+        if not config.get('use_pungi_in_bodhi'):
+
+            mash_cmd = 'mash -o {outputdir} -c {config} -f {compsfile} {tag}'
+            mash_conf = config.get('mash_conf')
+            if os.path.exists(previous):
+                mash_cmd += ' -p {}'.format(previous)
+            self.mash_cmd = mash_cmd.format(outputdir=outputdir, config=mash_conf,
+                                            compsfile=comps, tag=self.tag).split()
+
+        else:  # We are going to use pungi in this run
+            pungi_cmd = "pungi-koji  --config={config} --old-composes={outputdir}"
```

Comment:
@ralphbean One major advantage of shelling out is that we can continue to create the updateinfo.xml, since otherwise you would be getting into the GIL.